### ISSUE TITLE
Product Manual Library [Mobile]: (From QA) Adjust white space

### ISSUE THREAD

**Issue by MartyBeckers** [nickpaventi/culligan-diy#118]
http://dev.culligandiy.com/Product-Manual-Library: There is too much white space in the following instances:
a.	After the last product in a category
b.	When there is a category that only has one item 

![image](https://cloud.githubusercontent.com/assets/11283011/7091599/aed4b646-df5f-11e4-95a8-834c67e47322.png)

**Comment by nickpaventi** [comment]
@MartyBeckers This may be a content entry issue, there are empty columns with empty ULs. Can we exclude these in the loop? Are there blank entries in the CMS?
![screen shot 2015-04-10 at 11 53 01 am](https://cloud.githubusercontent.com/assets/10563172/7094674/4ab1da90-df78-11e4-9af3-54bdf21e81e1.png)

**Comment by MartyBeckers** [comment]
The original HTML/CSS was created for the mockup at http://invis.io/MZ2H76PQ5, then because all of the categories changed and required more than just three sections, the new mockup at http://invis.io/PQ2LHEKGD was created and that's the structure we integrated into Kentico.  In order to utilize the existing HTML/CSS with the new layout, we had to include empty columns to finish off each row (otherwise the products in each final row would snap to the right).  If you want to provide new HTML/CSS to resolve both the desktop and mobile issues, I can remove the blank entries.
